Unable to build a docker image

**Describe the bug**
Unable to build a docker image with the command `make docker`.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to root of the project with `cd <path to the root of the project>`
2. Run `make`
3. Run `make docker`
4. See error

**Expected behavior**
A docker image should be created.

**Screenshots**
<img width="1767" height="241" alt="Image" src="https://github.com/user-attachments/assets/4e3c52ac-dcfd-49d8-a20a-4fcb873edd4c" />

**Error in text**
```
Checking dependencies
Building minio binary to './minio'
Building minio docker image 'alwindoss/minio:RELEASE.2025-10-15T17-29-55Z-14-g71017be8a'
Dockerfile:10
--------------------
   8 |     COPY ./minio-${TARGETARCH}.${RELEASE} /usr/bin/minio
   9 |     COPY ./minio-${TARGETARCH}.${RELEASE}.minisig /usr/bin/minio.minisig
  10 | >>> COPY ./minio-${TARGETARCH}.${RELEASE}.sha256sum /usr/bin/minio.sha256sum
  11 |     
  12 |     COPY dockerscripts/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
--------------------
ERROR: failed to build: failed to solve: failed to compute cache key: failed to calculate checksum of ref 19fa3c52-76ae-409f-a23d-3c7ba1e5db6e::n7wtfi125s55irpyld1sgljg5: "/minio-amd64..sha256sum": not found
make: *** [Makefile:216: docker] Error 1

```

**go env output:**
```
AR='ar'
CC='gcc'
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_ENABLED='1'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
CXX='g++'
GCCGO='gccgo'
GO111MODULE=''
GOAMD64='v1'
GOARCH='amd64'
GOAUTH='netrc'
GOBIN='/home/alwin/Sandbox/opt/gohome/bin'
GOCACHE='/home/alwin/.cache/go-build'
GOCACHEPROG=''
GODEBUG=''
GOENV='/home/alwin/.config/go/env'
GOEXE=''
GOEXPERIMENT=''
GOFIPS140='off'
GOFLAGS=''
GOGCCFLAGS='-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -ffile-prefix-map=/tmp/go-build900989737=/tmp/go-build -gno-record-gcc-switches'
GOHOSTARCH='amd64'
GOHOSTOS='linux'
GOINSECURE=''
GOMOD='/home/alwin/Sandbox/src/jumuia/go.mod'
GOMODCACHE='/home/alwin/Sandbox/opt/gohome/pkg/mod'
GONOPROXY=''
GONOSUMDB=''
GOOS='linux'
GOPATH='/home/alwin/Sandbox/opt/gohome'
GOPRIVATE=''
GOPROXY='https://proxy.golang.org,direct'
GOROOT='/usr/local/go'
GOSUMDB='sum.golang.org'
GOTELEMETRY='local'
GOTELEMETRYDIR='/home/alwin/.config/go/telemetry'
GOTMPDIR=''
GOTOOLCHAIN='auto'
GOTOOLDIR='/usr/local/go/pkg/tool/linux_amd64'
GOVCS=''
GOVERSION='go1.25.4'
GOWORK=''
PKG_CONFIG='pkg-config'
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,18 +1,13 @@
-FROM minio/minio:latest
+FROM golang:latest AS builder
+WORKDIR /go/src/app
+COPY . .
+RUN make
 
-ARG TARGETARCH
-ARG RELEASE
-
-RUN chmod -R 777 /usr/bin
-
-COPY ./minio-${TARGETARCH}.${RELEASE} /usr/bin/minio
-COPY ./minio-${TARGETARCH}.${RELEASE}.minisig /usr/bin/minio.minisig
-COPY ./minio-${TARGETARCH}.${RELEASE}.sha256sum /usr/bin/minio.sha256sum
-
-COPY dockerscripts/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
-
-ENTRYPOINT ["/usr/bin/docker-entrypoint.sh"]
-
+# https://github.com/GoogleContainerTools/distroless
+FROM gcr.io/distroless/static-debian12
+WORKDIR /root/
+COPY --from=builder /go/src/app/minio .
+# COPY --from=builder dockerscripts/docker-entrypoint.sh ./docker-entrypoint.sh
+# ENTRYPOINT ["./docker-entrypoint.sh"]
 VOLUME ["/data"]
-
-CMD ["minio"]
+CMD [ "./minio", "server", "/data" ]

